src/components/views/game/room/RoomInfoContainer.tsx

```diff
@@ -5,15 +5,23 @@ import Button from "@mui/material/Button"
 import ContentCopyIcon from "@mui/icons-material/ContentCopy"
 
 export const RoomInfoContainer = observer(() => {
-  const { game, activeParticipations } = useCurrentGame()
+  const { game, activeParticipations, id } = useCurrentGame()
   const playerCount = activeParticipations?.length || 0
+  const inviteLink = `localhost:3000/joinGame/${id}`
 
   return (
     <div className={`room-info-container`}>
       <h2 className="infoPanel"> Room Info </h2>
       <div> This is room: {game.name}</div>
       <div> Current player count: {playerCount} </div>
-      <Button variant="contained" endIcon={<ContentCopyIcon />}>
+      <Button
+        //TODO Implement a way that the link (heroku vs localhost) is not hardcoded
```

Comment:
export getDomain of api.ts

Reply:
bekomme ich da nicht das backend? also 8080 order -server?

Reply:
true.
Then i would use useLocation(), maybe there you find host, protocol and port (thats what you need).
Or just window.location..., it should be there.

Reply:
This worked, thank you!